src/preact/storybook/Hooks.js

```diff
@@ -7,16 +7,22 @@ export default {
   title: '0/Hooks',
 };
 
+/**
+ * @param {{prop: *}} param0
+ * @return {import('preact').VNode}
+ */
 function Component({prop}) {
   const [text, setText] = useState('initial render');
   const ioCallback = useCallback(
+    /** @param {{isIntersecting: boolean}} param0 */
     ({isIntersecting}) => {
       setText(`is intersecting for ${prop}: ${isIntersecting}`);
     },
     [prop]
   );
 
   const anotherIoCallback = useCallback(
+    /** @param {{isIntersecting: boolean}} param0 */
```

Comment:
Do this and the `{{prop: *}}` both represent some common type?

Reply:
Nope! This an an InOb record. I have no idea what prop is meant to be. Cast to `any` and figure it can be figured out by someone else down the line.